src/components/Tokens/TokenDetails/PriceChart.tsx

```diff
@@ -148,6 +175,34 @@ export function PriceChart({ width, height, prices: originalPrices, timePeriod }
     )
   ) : null
 
+  const tooltipMessage = (
+    <>
+      <Trans>Price may not be up-to-date due to recently low trading volume</Trans>
```

Comment:
This reads quite awkwardly imo.

```suggestion
      <Trans>This price may not be up-to-date due to low trading volume.</Trans>
```